### PR TITLE
Final pull for project

### PR DESCRIPTION
**Pull request by davehannon**
So long, and thanks for all the fish.  ;)

### CODE REVIEW

File: app/lib/controllers/api.js [hunk 1]
```diff
@@ -12,9 +12,17 @@ const version = require('../../package.json').version
 const requestPromise = require('request-promise')
 const auth = require('../middleware/auth')
 const isHUDHQUser = auth.isHUDHQUser
+const path = require('path')
 
 const validLocaleTypes = ['city', 'county', 'congrdist', 'zipcode', 'township', 'tract']
 
+if (process.env.DRDP_LOCAL === true || process.env.DRDP_LOCAL === 'true') {
```
Comment (waldoj) [inline app/lib/controllers/api.js]: Relatable. 😬

**Comment by waldoj** [comment]
✓ Read all changes in code
✓ Extra-carefully reviewed all changes in tests and new tests
✓ Ran all tests (final code coverage tally: 93.29%)
✓ Ran the application locally, performed manual testing of functionality

I've encountered just one problem, which is that the `Export` button is largely hidden in the smallest breakpoint and the second-widest breakpoint. That's a result of the `Download Data Dictionary` being added, pushing the `Clear` and `Export` buttons over. Once that bug is cleared up, I'm good to merge this pull request.

**Comment by davehannon** [comment]
Paul wanted to go with this, losing the non-working Feedback link, and adding the Data Dictionary link.

**Comment by waldoj** [comment]
A fine solution. Approved.

**Pull request merged**